src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -393,19 +393,6 @@ async function pullMessages() {
 
   return [subClient, response.receivedMessages] as const;
 }
-
-async function run() {
-  const [subClient, receivedMessages] = await pullMessages();
-  await poll(subClient, receivedMessages ?? []);
-
-  // We're transitioning this into a long-running process,
-  // but to ensure a smooth transition, allow keeping the cronjob behaviour:
-  if (process.env.LOOP === "true") {
```

Comment:
You mentioned in the PR top-level comment that this wasn't necessary to resolve the issue, do we still want this behavior? I thought this was moved from a cron job to a long-running task, is there a shell script or something wrapping this?

Just FYI, I believe that Node will eventually run out of stack space running recursively like this, you probably want a `while` or other type of iterative loop (because JS doesn't have tail call optimization).

Reply:
As I understand it, [Brandon never actually activated](https://mozilla.slack.com/archives/C08C8P5RTBN/p1741106222206889?thread_ts=1741027363.416669&cid=C08C8P5RTBN) the long-running task, since handling the SIGINT by itself was enough to resolve the issue. So that's why I removed it.

(And good call about the stack issue, I didn't think of that.)